ipv6/ipv6.go

```diff
@@ -66,6 +78,52 @@ var _ = IPv6Describe("IPv6 Connectivity Tests", func() {
 		}
 	}
 
+	describeIpv6NginxTest := func(assetPath, stack string) {
+		commandOptions := []string{"push", appName, "-s", stack, "-p", assetPath, "-m", DEFAULT_MEMORY_LIMIT}
+		defaultPathExpectMessage := "Hello"
+		pushSession := cf.Cf(commandOptions...)
+		exitCode := pushSession.Wait(Config.DetectTimeoutDuration()).ExitCode()
+
+		// Log output if app push fails to diagnose deployment errors. This is crucial for
+		// identifying issues such as unsupported IPv6, which prevent the Nginx app from
+		// being successfully deployed. Capturing stdout and stderr provides insights into
+		// configuration or connectivity problems.
+
+		if exitCode != 0 {
```

Comment:
The default pattern we have for testing a `cf push` is:
```
					Expect(cf.Cf("push",
						appName
					).Wait(Config.DetectTimeoutDuration())).To(Exit(0))
```
And then checking the app's outputs in an `AfterEach` function:
```
	AfterEach(func() {
		app_helpers.AppReport(appName)

		Expect(cf.Cf("delete", appName, "-f", "-r").Wait()).To(Exit(0))
	})
```

E.g. https://github.com/cloudfoundry/cf-acceptance-tests/blob/cd5c27ef81e7e81dfa32f99b5aa28e8b2fed5506/detect/buildpacks.go#L29

Can you apply this pattern?